pkg/apiserver/vshn/nextcloud/backup.go

```diff
@@ -37,6 +38,15 @@ func New() restbuilder.ResourceHandlerProvider {
 			return nil, err
 		}
 
+		cache, err := ristretto.NewCache(&ristretto.Config[string, []byte]{
```

Comment:
This call is duplicated several times. It might make more sense to keep it in the common.go file.

Reply:
Fixed